activerecord/test/models/company.rb

```diff
@@ -10,7 +10,6 @@ class Company < AbstractCompany
   has_one :dummy_account, :foreign_key => "firm_id", :class_name => "Account"
   has_many :contracts
   has_many :developers, :through => :contracts
-  has_many :accounts
```

Comment:
@pixeltrix 
Not only that, but also because `Account` is defined as:
```ruby
  create_table :accounts, force: true do |t|
    t.integer :firm_id
    t.string  :firm_name
    t.integer :credit_limit
  end
```
So, `company.accounts` fails because `accounts` doesn't have `company_id`. ITOH, `firm.accounts` works because `accounts` has `firm_id`